src/fairseq2/recipe/config.py

```diff
@@ -908,16 +940,25 @@ def validate(self) -> ValidationResult:
 
 @dataclass(kw_only=True)
 class TriStageLRConfig(Validatable):
+    """Represents the configuration of :class:`TriStageLRConfig`."""
```

Comment:
Did you want to put:
```suggestion
    """Represents the configuration of :class:`TriStageLR`."""
```

Reply:
Oh good catch, thanks!